pkgs/development/libraries/pcre/default.nix

```diff
@@ -27,6 +27,7 @@ in stdenv.mkDerivation {
   configureFlags = optional (!stdenv.hostPlatform.isRiscV && !(stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isAarch64)) "--enable-jit" ++ [
     "--enable-unicode-properties"
```

Comment:
According to the docs here https://www.pcre.org/original/doc/html/pcrebuild.html#SEC7 this option already enforces ``--enable-utf8``. I think we can close this.